x509util/name.go

```diff
@@ -35,7 +53,7 @@ func newName(n pkix.Name) Name {
 		PostalCode:         n.PostalCode,
 		SerialNumber:       n.SerialNumber,
 		CommonName:         n.CommonName,
-		ExtraNames:         newDistinguisedNames(n.ExtraNames),
+		ExtraNames:         newDistinguisedNames(n.Names),
```

Comment:
I think you kept the `ExtraNames` property for backwards compatibility? Should we maybe also provide the `Names` property, which could contain the same data?